proposals/NNNN-New-vehicle-data-StabilityControlsStatus.md

```diff
@@ -0,0 +1,172 @@
+# New vehicle data StabilityControlsStatus
+
+* Proposal: [SDL-NNNN](NNNN-New-vehicle-data-StabilityControlsStatus.md)
+* Author: [Ankur Tiwari](https://github.com/atiwari9)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core | HMI | Policy Server | SHAID | iOS | Java Suite]
+
+## Introduction
+
+This proposal is to add new vehicle data `StabilityControlsStatus` to know if stability controls like ESC, Traction control etc. are active.
+
+## Motivation
+
+In order to partner with more diverse app partners, we need to provide additional sets of vehicle data items through SDL. This goes in line with commitment to enhance SDL with even richer vehicle data content.
```

Comment:
Can this be expanded upon to be more specific as to why this particular vehicle data item is proposed?

Reply:
Done.